client/components/site-selector/index.jsx

```diff
@@ -393,6 +393,7 @@ const SiteSelector = React.createClass( {
 				<Search
 					ref="siteSearch"
 					onSearch={ this.onSearch }
+					delaySearch={ true }
```

Comment:
Just to be clear, which "it"? Having the delay for everyone, or only having the delay when you have more than some number of sites?